src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProjectSteps.java

```diff
@@ -0,0 +1,110 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.fundacionjala.core.selenium.WebDriverManager;
+import org.fundacionjala.pivotal.context.ProjectContext;
+import org.fundacionjala.pivotal.entities.Project;
+import org.fundacionjala.pivotal.ui.pages.LoggedIn.DashboardPage;
+import org.fundacionjala.pivotal.ui.pages.LoggedIn.ProjectPage;
+import org.fundacionjala.pivotal.ui.pages.LoggedIn.ProjectsSummaryPage;
+import org.fundacionjala.pivotal.ui.popups.CreateProjectPopup;
+import org.testng.asserts.SoftAssert;
+import static org.junit.Assert.assertTrue;
+import java.util.Map;
+
+public class ProjectSteps {
+
+    // Pages
+    private DashboardPage dashboardPage;
+    private CreateProjectPopup createProjectPopup;
+    private ProjectPage projectPage;
+    private ProjectsSummaryPage projectsSummaryPage;
+
+    // Entity
+    private Project project;
+
+    //Context
+    private final ProjectContext projectContext;
+
+    private static final int WAIT_TIME = 3000;
+    /**
+     * Adding Dependency injection to share Project Context information.
+     * @param sharedProjectContext
+     */
+    public ProjectSteps(final ProjectContext sharedProjectContext) {
+        this.projectContext = sharedProjectContext;
+    }
+
+    /**
+     * StepDef to open the Create Project pop-up.
+     */
+    @When("I open the Create Project pop-up")
+    public void openTheCreateProjectPopUp() {
+        dashboardPage = new DashboardPage();
+        createProjectPopup = dashboardPage.goToProjectCreation();
+    }
+
+    /**
+     * StepDef to create a new public project with the provided data.
+     * @param projectInfo
+     */
+    @When("I create a new public Project with the following information")
+    public void createNewProject(final Map<String, String> projectInfo) {
+        //Updating Project entity
+        project = new Project();
+        project.setName(projectInfo.get("Name"));
+        project.setAccount(projectInfo.get("Account"));
+        project.setPrivacy(projectInfo.get("Privacy"));
+
+        //Creating Project from UI
+        projectPage = createProjectPopup.createPublicProject(projectInfo.get("Name"), projectInfo.get("Account"));
+    }
+
+    /**
+     * StepDef to check that I was driven to recently created Project Page.
+     */
+    @Then("I am driven to recently created Project Page")
+    public void verifyIAmDrivenToProjectPage() {
+        try {
+            Thread.sleep(WAIT_TIME);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        } finally {
+            String currentUrl = WebDriverManager.getInstance().getCurrentUrl();
+            assertTrue(currentUrl.startsWith("https://www.pivotaltracker.com/n/projects/"));
```

Comment:
This verifications are unnecessary, if you want to make sure you are in the page you can verify the information of the project